.vsts.template.nonwindows.yml

```diff
@@ -18,6 +19,8 @@ steps:
 - script: ./dev.sh layout Release
   workingDirectory: src
   displayName: Build & Layout Release
+  env:
```

Comment:
Could you give an example of this working?

Reply:
https://dev.azure.com/mseng/PipelineTools/_build/results?buildId=11504349&view=logs&j=5b4209a6-3453-5624-9b5c-f3d75dcd52a3&t=06a06f86-759a-5666-a10d-c4678e42ef5f